### DISCUSSION TITLE
Fdroid install auto supplanted by Google Play Store

### DISCUSSION THREAD

**Discussion by habere-et-dispertire** [tutao/tutanota#4347] [General]
Google Play Store has summarily usurped and upgraded my Fdroid installed client. Is this of concern?

There is a clear separation of concerns in the different software repositories that is not being respected, but beyond that I imagine non-trivial issues arising from doing this.

**Comment by charlag** [comment]
You can't install one over another, they are signed with different keys so Android wouldn't allow you to do that
